www/__mocks__/cordovaMocks.ts

```diff
@@ -100,11 +101,22 @@ export const mockBEMUserCache = () => {
       );
     },
     getDocument: (key: string, withMetadata?: boolean) => {
-      return new Promise<any[]>((rs, rj) =>
-        setTimeout(() => {
-          rs(_storage[key]);
-        }, 100),
-      );
+      // this was mocked specifically for enketoHelper's use, could be expanded if needed
+      const fakeSurveyConfig = fakeConfig;
```

Comment:
One way of expanding the way we mock configs might be to accept the fakeConfig as an optional parameter of `mockCordova`. That way, we specify what config to use *from the test*, not from this shared code.

Reply:
Because the way it is right now, using fakeSurveyConfig affects all tests, even ones that this config might not make sense for.

Reply:
The parameter option would also help me test more cases, right? I think that sounds like a good plan

Reply:
I just added the parameter option in [22decbf](https://github.com/e-mission/e-mission-phone/pull/1063/commits/22decbfd0b58776450737ba536dc9bf42e375aa5), running smoothly for now! Going to see if I can leverage this to add some cases to `resolveLabel` tests

Reply:
I was able to add more test cases in [80aa5ea](https://github.com/e-mission/e-mission-phone/pull/1063/commits/80aa5ea18081849819236d65dc5d07ea583aa8e8)!